app/src/main/java/uk/gov/register/Field.java

```diff
@@ -0,0 +1,43 @@
+package uk.gov.register;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.register.datatype.Datatype;
+import uk.gov.register.datatype.DatatypeFactory;
+
+import java.util.Optional;
+
+public class Field {
+    final String fieldName;
+    final Datatype datatype;
+    final Optional<String> register;
+    final Cardinality cardinality;
```

Comment:
do we need `phase` here now? (or at least, to mark it as a `@JsonIgnoreProperty`?)